recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -10,25 +10,43 @@ OECMAKE_SOURCEPATH = "${S}/"
 
 SECTION = "net"
 LICENSE = "BSD-2-Clause-Patent.txt"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
-DEPENDS = "python-native python-pyyaml-native json-c openssl zeromq"
+# LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=0518d409dae93098cca8dfa932f3ab1b"
+# DEPENDS = "python-native python-pyyaml-native json-c openssl nanomsg"
+# DEPENDS = "python-native python-pyyaml-native json-c openssl wpa-supplicant libnl"
+DEPENDS = "python-native python-pyyaml-native json-c openssl nng wpa-supplicant libnl"
 RDEPENDS_${PN} = "iproute2 busybox readline"
 SUMMARY = "prplMesh"
 
-SRCREV = "911392026280d9961823a9359e6cc4f08c9ee0a2"
-SRC_URI = "git://github.com/prplfoundation/prplMesh.git;protocol=http;branch=master"
+# SRCREV = "911392026280d9961823a9359e6cc4f08c9ee0a2"
+SRCREV = "d9eb170b69c1c225e40df5c191e8978568ec2e08"
+SRC_URI = "git://github.com/prplfoundation/prplMesh.git;protocol=http;branch=master \
+           file://001-add_find_nl_for_nl80211_bwl_type.patch"
 
 BUILD_TYPE ?= "Release"
 TARGET_PLATFORM ?= "linux"
-BWL_TYPE ?= "DUMMY"
+# BWL_TYPE ?= "DUMMY"
+# BWL_TYPE ?= "DWPAL"
+BWL_TYPE ?= "NL80211"
 INSTALL_PREFIX ?= "/opt/prplmesh"
 
 EXTRA_OECMAKE = "-DTARGET_PLATFORM=${TARGET_PLATFORM} \
                  -DBWL_TYPE=${BWL_TYPE} \
                  -DUSE_USER_TMP_PATH=0 \
                  -DCMAKE_INSTALL_PREFIX:PATH=${INSTALL_PREFIX} \
                  -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
-                 -DMSGLIB=zmq"
+                 -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
```

Comment:
This is a bit weird to have in a production build...

Reply:
Actually maximal verbosity is sane configuration for build systems like Yocto: By default all logs hided by yocto itself, but in case of issues verbose build allow to get information what exactly goes wrong.